package.json

```diff
@@ -44,36 +44,31 @@
     "babel-cli": "6.18.0",
     "babel-core": "6.20.0",
     "babel-eslint": "7.1.1",
+    "babel-jest": "^18.0.0",
```

Comment:
A few of these have carrots. I've leaned toward specifying exact version for consistency. But I could be swayed. Either way, we should handle all these the same.